wifi: Minor TWT fixes

krish2718 opened the pull request.

krish2718 committed:
net: wifi: Fix a typo

Typo in TWT error strings.

Signed-off-by: Chaitanya Tata <Chaitanya.Tata@nordicsemi.no>

krish2718 committed:
net: wifi: Do TWT checks only for setup

For tear down the checks doesn't make any sense, as TWT flow is already
established.

Signed-off-by: Chaitanya Tata <Chaitanya.Tata@nordicsemi.no>

krish2718 committed:
net: wifi: Disable trigger in TWT quick setup

Most AP's are not sending proper HE triggers or stopping triggers after
sometime, so, change the default to non-triggered based TWT.

Signed-off-by: Chaitanya Tata <Chaitanya.Tata@nordicsemi.no>

rlubos commented on subsys/net/l2/wifi/wifi_shell.c:
```diff
@@ -717,7 +717,7 @@ static int cmd_wifi_twt_setup_quick(const struct shell *sh, size_t argc,
 	params.setup.responder = 0;
 	params.setup.implicit = 1;
-	params.setup.trigger = 1;
+	params.setup.trigger = 0;
```
Please change the commit title, `Seeing issues with trigger enabled TWT` is not really useful for understanding the change, just mention you change the default.

krish2718 committed:
net: wifi: Add an error code for TWT duplicate flow request

This is to handle duplicate TWT setup request.

Signed-off-by: Chaitanya Tata <Chaitanya.Tata@nordicsemi.no>

krish2718 committed:
net: wifi: Fix tear down messages

TWT tear down is a synchronous call, so, once it returns the operation
is done.

Signed-off-by: Chaitanya Tata <Chaitanya.Tata@nordicsemi.no>

krish2718 committed:
net: wifi: Add a demarcation print

This helps see the response status and details separately.

Signed-off-by: Chaitanya Tata <Chaitanya.Tata@nordicsemi.no>

krish2718 committed:
net: wifi: Add a message for unsolicited TWT tear down

This is for TWT tear down by AP.

Signed-off-by: Chaitanya Tata <Chaitanya.Tata@nordicsemi.no>
